Source/JavaScriptCore/inspector/scripts/codegen/models.py

```diff
@@ -429,7 +429,8 @@ def parse_domain(self, json, isSupplemental):
                 raise ParseException("Malformed domain specification: events is not an array")
             events.extend([self.parse_event(event, debuggable_types) for event in json['events']])
 
-        self.domains.append(Domain(json['domain'], json.get('description', ''), json.get('condition'), debuggable_types, target_types, isSupplemental, version, types, commands, events))
+        domain_name_exposed_as = json.get('exposedAs', json['domain'])
```

Comment:
should this be `domainExposedAs`?

Reply:
Ah I see, I wrote the commit message in a hurry. I actually prefer 'exposedAs' to 'domainExposedAs'. How about you?